internal/command/platform/status.go

```diff
@@ -2,31 +2,135 @@ package platform
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+	"time"
 
 	"github.com/skratchdot/open-golang/open"
 	"github.com/spf13/cobra"
 
+	"github.com/superfly/flyctl/api"
 	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/config"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/flyctl/internal/logger"
+	"github.com/superfly/flyctl/internal/render"
 	"github.com/superfly/flyctl/iostreams"
 )
 
 func newStatus() (cmd *cobra.Command) {
 	const (
-		long = `Show current Fly platform status in a browser
+		long = `Show current Fly platform status in a browser or via json with the json flag
 `
-		short = "Show current platform status"
+		short = "Show current platform status with an optional filter for maintenance or incidents in json mode (eg. incidents, maintenance)"
 	)
 
-	cmd = command.New("status", short, long, runStatus)
+	cmd = command.New("status [kind](optional)", short, long, runStatus)
```

Comment:
Sorry for the nitpick but in [BNF grammar](https://en.wikipedia.org/wiki/Extended_Backus%E2%80%93Naur_form) the square brackets mean the parameter is optional, somehow I found the extra `(optional)` text confusing but it could be just me.

Reply:
til, I wasn't sure if the square brackets meant optional